### ISSUE TITLE
NTR: Modifier: Occipital localization to lymphadenopathy (Face & neck)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7824]
New term request Modifier: Occipital localization to lymphadenopathy (seidel_190324162326): Posterior scalp; occiput; includes postauricular lymphadenopathy; LAD  UMLS: 0920024

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000175
- label: Occipital location
- parents: [Localized (HP:0012838)](https://hpo.jax.org/app/browse/term/HP:0012838)
- pmid: 
- def: Applies to an abnormality that is situated in the occiput (back part of the head).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7824
- synonyms:
